FullyNoded/View Controllers/Home/Home Screen/MainMenuViewController.swift

```diff
@@ -85,6 +85,7 @@ class MainMenuViewController: UIViewController, UITableViewDelegate, UITableView
     
     func torConnProgress(_ progress: Int) {
         print("progress = \(progress)")
+        displayAlert(viewController: self, isError: false, message: "Tor progress: \(progress)%")
```

Comment:
not so much of a fan of this approach as it presents a new UILabel every time the progress updates, it would be much better to update one label with dynamic text instead of presenting what could be ~20 new labels in rapid fire